src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -125,4 +125,16 @@ public void setTankDrivePowers(double leftScale, double rightScale) {
   private double squareInput(double value) {
     return Math.copySign(value * value, value);
   }
+
+  /**
+   * Drive the system with the given voltage values for each side of the drivetrain.
+   *  
+   * @param left left motor voltages
+   * @param right right motor voltages
+   */
+  public void setMotorVoltages(double left, double right) {
+    // TODO: stub on purpose, see comment in FollowPathCommand
+    // This doesn't really match our current interface, but neither does the other option;
+    // Should a state on this subsystem be added to switch between voltage and percent output?
```

Comment:
Alternatively, instead of `tankSubsystem` containing states and continually setting the motor controller output in `periodic`, the motor controllers can be the ones to store the state and methods can just set their values. Instead of
```java
private int leftPower;
private int rightPower;

public void setTankDrivePowers(double leftScale, double rightScale) {
  this.leftPower = leftScale;
  this.rightPower = rightScale;
}

@Override
public void periodic() {
  // These must be in percent output mode
  leftMain.set(leftPower);
  rightMain.set(rightPower);
}
```
why not have
```java
public void setTankDrivePowers(double leftScale, double rightScale) {
  leftMain.set(leftScale);
  rightMain.set(rightScale);
}
```
? This would also allow for more flexibility with the output mode, such as
```java
public void setTankDrivePowers(double leftScale, double rightScale) {
  leftMain.set(leftScale);
  rightMain.set(rightScale);
}

public void setTankDriveVoltages(double leftVoltage, double rightVoltage) {
  leftMain.setVoltage(leftVoltage);
  rightMain.setVoltage(rightVoltage);
}
```

Reply:
Allowing voltages would also resolve 1/2 of the current problems with the path following command.